### PR TITLE
Divide README into several docs

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Part of #274 

@nikitin-da PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,312 +2,155 @@
 
 #####Overview:
 * Powerful & Simple set of Operations: `Put`, `Get`, `Delete`
-* Convenient builders with compile-time guarantees. Forget about 6-7 `null` in queries
+* API for Humans: Type Safety, Immutability & Thread-Safety
```
Comment (nikitin-da) [inline README.md]: =)

**Comment by artem-zinnatullin** [comment]
Updated

**Comment by nikitin-da** [comment]
I will place it here for few days
:octocat:

**Comment by artem-zinnatullin** [comment]
What are you talking about?

**Pull request merged**